catala/catala.md

```diff
@@ -6,4 +6,8 @@ Tanque els ulls els torne a obrir. No pot ser. On estic?
 
 Em resulta familiar, però no sé on estic. [Que és aixó ](http://www.google.es/imgres?imgurl=http%3A%2F%2F1.bp.blogspot.com%2F-b0CZu7RAyiM%2FULaNOLTJMZI%2FAAAAAAAAAA8%2FwmVTGAKDC-U%2Fs400%2FCaptura3.PNG&imgrefurl=http%3A%2F%2Fsusandsan.blogspot.com%2F2012%2F11%2Fsimbolismo-figuracion-y-abstraccion.html&h=304&w=400&tbnid=jAdgGBhRlYxgwM%3A&zoom=1&docid=j7NOlpg8lyjPMM&ei=xo95VOvhDc7zasWGgaAP&tbm=isch&ved=0CGMQMygfMB8&iact=rc&uact=3&dur=1931&page=3&start=20&ndsp=13)
 
-Que fas?
\ No newline at end of file
+Que fas?
+
+Vas a mirar-te al mirall. Que veus?
+
+Em sembla que em falta alguna cosa a la cara.. on es la meva [orella](http://la.guestofaguest.com/wp-content/uploads/2010/10/ear.jpg)?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.